3회차/오채현/3_파리퇴치.py

```diff
@@ -1,3 +1,20 @@
 import sys
 
 sys.stdin = open("_파리퇴치.txt")
+
+T = int(input())
+
+for testcase in range(1, T + 1):
+    # for testcase in range(1):
+    n, m = map(int, input().split())
+    matrix = [list(map(int, input().split())) for _ in range(n)]
+    maxT = 0
+
+    for i in range(n - m + 1):
+        for j in range(n - m + 1):
+            total = 0
+            for k in range(m):
+                total += sum(matrix[i+k][j:j+m])
+                if total > maxT:
+                    maxT = total
+    print(f'#{testcase} {maxT}')
```

Comment:
멋진 풀이네요! 👍 
`total`을 다 더한 후 확인해줘도 좋을 것 같습니다 😄 
```suggestion
            total = 0
            for k in range(m):
                total += sum(matrix[i+k][j:j+m])
            if total > maxT:
                maxT = total
    print(f'#{testcase} {maxT}')
```